flow/layers/platform_view_layer.cc

```diff
@@ -29,6 +29,7 @@ void PlatformViewLayer::Preroll(PrerollContext* context,
                                            context->display_list_enabled);
   context->view_embedder->PrerollCompositeEmbeddedView(view_id_,
                                                        std::move(params));
+  context->view_embedder->PushVisitedPlatformView(view_id_);
```

Comment:
Platform view is pushed but nothing ever clears that list, so it will have the same issue as the filters, it slows down rendering until app is unresponsive.

Reply:
Yes, we are fixing it here: https://github.com/flutter/engine/pull/34596